src/types/constants.d.ts

```diff
@@ -6,7 +6,6 @@ import { Scalar } from './either';
 export const null_: Decoder<null>;
 export const undefined_: Decoder<undefined>;
 export function constant<T extends Scalar>(value: T): Decoder<T>;
-export function constant<T>(value: T): Decoder<T>;
```

Comment:
I originally left this, on the off chance that anybody was using `constant()` with something odd like an Object or Symbol (which wouldn't be covered by the `Scalar` type).

Reply:
Yeah I appreciate that!

I decided to remove it after all, because I've never seen any real use of `constant()` it other than with scalars, which matches its intention. I figured we might as well express the intent clearly here, and avoid cases where users may be surprised by its behavior if you would feed it an object instance. This would never match a value in practice, unless it's somewhat fabricated, I guess.

Symbols are another interesting example indeed, which I considered including in the Scalar definition originally, if only this was a bit harder to keep in sync between Flow and TypeScript, so I decided to keep them off for now and retry adding those in later. I'm not at all opposed to add symbols back to the scalar definition again, if there is a need for it.

Thanks again!

Reply:
Good point!  

TypeScript's `Symbol` support still seems pretty shaky, so I can't blame you for taking a "wait and see" approach.

Reply:
Same for Flow's! Eventually it will happen :)